docs/production/authentication-methods.md

```diff
@@ -405,8 +405,9 @@ it as follows:
     1. On your Zulip server, `mkdir -p /etc/zulip/saml/idps/`
     2. Put the IDP public certificate in `/etc/zulip/saml/idps/{idp_name}.crt`
     3. (Optional) Put the Zulip server public certificate in `/etc/zulip/saml/zulip-cert.crt`
-    4. (Optional) Put the Zulip server private key in `/etc/zulip/saml/zulip-private-key.key`
-    5. Set the proper permissions on these files and directories:
+       and the corresponding private key in `/etc/zulip/saml/zulip-private-key.key`. Note that
+       the certificate is supposed to be a standard X.509 certificate.
```

Comment:
The confusion in #17955 was if it wanted a full cert chain, or just a single cert, which seems like we should call out; e.g. something like

```suggestion
       and the corresponding private key in `/etc/zulip/saml/zulip-private-key.key`. Note that
       the certificate should be the single X.509 certificate for the server, not a full chain of
       trust, which consists of multiple certificates.
```